xml/ha_concepts.xml

```diff
@@ -742,9 +742,9 @@
     Based on the information in the CIB, the PE then computes the ideal
     state of the cluster and how it should be achieved. The PE feeds a list of
```

Comment:
Figured there're references of "PE" which stands for "pengine" :-) Now it's renamed to pacemaker-schedulerd.

Reply:
Yan, does this mean that we need to change any instance of PE or policy engine in the docs? I was of the impression that only the daemon name changed, but that 'policy engine' as a component is still a valid term that we can continue to use. See also https://www.suse.com/documentation/sle-ha-15/book_sleha_guide/data/sec_ha_architecture.html#sec_ha_architecture_layers.

Reply:
Indeed the old terms are still used here and there, but eventually we'd probably better use the consistent names avoiding confusions :-)
https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html/Pacemaker_Explained/_pacemaker_architecture.html

Reply:
Ok, I will remove PE and policy engine from our docs then. This means I also need to adjust the graphics (and the overview text).

Reply:
In that case, we'd probably better as well rename the other components:

CRM (Cluster Resource Manager) -> pacemaker-controld
LRM (Local Resource Manager) -> pacemaker-execd

Reply:
@gao-yan: I will address all of the above in this comment in a separate PR.